src/SourceBuild/Arcade/src/Tarball_WriteSourceRepoProperties.cs

```diff
@@ -104,16 +104,8 @@ private static DerivedVersion GetVersionInfo(string version, string commitCount)
                 var releaseParts = nugetVersion.Release.Split('-', '.');
                 if (releaseParts.Length == 2)
                 {
-                    if (releaseParts[1].TrimStart('0') == commitCount)
-                    {
-                        // core-sdk does this - OfficialBuildId is only used for their fake package and not in anything shipped
-                        return new DerivedVersion { OfficialBuildId = DateTime.Now.ToString("yyyyMMdd.1"), PreReleaseVersionLabel = releaseParts[0] };
-                    }
-                    else
-                    {
-                        // NuGet does this - arbitrary build IDs
-                        return new DerivedVersion { OfficialBuildId = releaseParts[1], PreReleaseVersionLabel = releaseParts[0] };
-                    }
+                    // NuGet does this - arbitrary build IDs
+                    return new DerivedVersion { OfficialBuildId = DateTime.Now.ToString("yyyyMMdd.1"), PreReleaseVersionLabel = releaseParts[0] };
```

Comment:
I don't understand this change.  Does the comment apply here?

Reply:
From my inspection NuGet is the only one hitting this case.  Their build release build number is incremental and is currently a three digit number.  This doesn't align with the `yyyyMMdd.r` format arcade expects.  Prior to the sha update, nuget was falling into the 3 part case and just happened to work but wasn't functioning as expected.  I'm going to log a follow up issue to refactor this code to be repo specific instead of number of parts.  This refactoring is not required at this point in time.